.github/CODEOWNERS

```diff
@@ -1,5 +1,7 @@
 # See GitHub's docs for more details:
+
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 # TypeScript Team
-* @workos-inc/typescript
+
```

Comment:
VSCode keeps adding this line, just leaving it since it doesn't really affect anything.